config/config-builder/src/key_manager_config.rs

```diff
@@ -6,11 +6,11 @@ use libra_config::config::{KeyManagerConfig as KMConfig, SecureBackend, Token, V
 use libra_types::chain_id::ChainId;
 
 pub struct KeyManagerConfig {
-    pub chain_id: ChainId,
     pub rotation_period_secs: Option<u64>,
     pub sleep_period_secs: Option<u64>,
     pub txn_expiration_secs: Option<u64>,
 
+    pub chain_id: ChainId,
```

Comment:
why? :O

Reply:
The chain id has more in common with the json rpc interface (i.e., it helps to specify which blockchain we're pointing to). The previous grouping with the time periods seemed very odd to me! Although this is a clarification that I don't feel super strongly about..